packages/dialtone-vue2/components/rich_text_editor/extensions/mentions/MentionComponent.vue

```diff
@@ -5,7 +5,7 @@
   >
     <dt-link
       kind="inverted"
```

Comment:
Not sure why `inverted` is here. I'd view this `--mention` variant to be a new `kind` value since there wouldn't be a mention variant of `danger`, `warning`, etc.

Reply:
yeah.. I think that makes a lot more sense.